dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/sampling/DeterministicSamplerTest.kt

```diff
@@ -83,34 +86,33 @@ internal class DeterministicSamplerTest {
 
     @RepeatedTest(128)
     fun `the sampler will sample the values based on the fixed sample rate`(
-        @LongForgery traceIds: List<Long>,
         @FloatForgery(min = 0f, max = 100f) fakeSampleRate: Float
     ) {
         // Given
+        println("fakeTraceIds.size: ${fakeTraceIds.size}")
         whenever(mockSampleRateProvider.invoke()) doReturn fakeSampleRate
         var sampledIn = 0
 
         // When
-        traceIds.forEach {
+        fakeTraceIds.forEach {
             if (testedSampler.sample(it.toULong())) {
                 sampledIn++
             }
         }
 
         // Then
-        assertThat(sampledIn.toFloat()).isCloseTo(traceIds.size * fakeSampleRate / 100f, Offset.offset(7.5f))
+        val offset = 2.5f * fakeTraceIds.size
+        assertThat(sampledIn.toFloat()).isCloseTo(fakeTraceIds.size * fakeSampleRate / 100f, Offset.offset(offset))
```

Comment:
Note: 

This might reduce the chance of flakiness, but can't completely avoid the test fail in my opinion
Let's keep it this way and see if it still fails